src/platform/user/authentication/utilities.js

```diff
@@ -20,13 +20,14 @@ function sessionTypeUrl(
   version = 'v0',
   application = null,
   to = null,
+  queryParams = null,
 ) {
   const base =
     version === 'v1'
       ? `${environment.API_URL}/v1/sessions`
       : `${environment.API_URL}/sessions`;
 
-  const searchParams = new URLSearchParams();
+  const searchParams = new URLSearchParams(queryParams || {});
```

Comment:
With the default parameter set to `{}` instead of null the `|| {}` wouldn't be needed here.